index.js

```diff
@@ -55,7 +55,7 @@ module.exports = withConfigLoader(function (loadConfig) {
       var map
       var warnings = result.warnings().join('\n')
 
-      file.contents = new Buffer(result.css)
+      file.contents = new Buffer(result.content || result.css)
```

Comment:
According to http://api.postcss.org/Result.html `content` is just an alias to `css`. Do we really need it?